### ISSUE TITLE
qt types compatibility

### ISSUE THREAD

**Issue by ntfshard** [p-ranav/csv2#16]
Hello

Is it make sense to prepare pull request with Qt types support? (or at least try to do it)
I believe sfinae magic should be enough for this.
Or library should be stl compatible only? Asking just for saving our time)

**Comment by sonulohani** [comment]
What Qt types are you talking about here. As Qt is dependent on C++ so all types will work except std::string which anyway you can convert it to QString by using fromStdString method.

**Comment by ntfshard** [comment]
> What Qt types are you talking about here. As Qt is dependent on C++ so all types will work except std::string which anyway you can convert it to QString by using fromStdString method.

Yep, but it doesn't look elegant. Moreover I noticed some weird issues with non-ascii symbols in patch -- it leads to crash.

**Comment by sonulohani** [comment]
I dont know if the devs are active for this repositories.

**Comment by ntfshard** [comment]
Or use other from fffaraz/awesome-cpp#csv repo

**Comment by sonulohani** [comment]
> Or use other from fffaraz/awesome-cpp#csv repo

You could use other. But their could be subtle difference between these libraries. For fast parsing, csv2 is the best.
